frontend/src/components/InputForm.vue

```diff
@@ -461,6 +464,7 @@ export default {
     employeeCountRange: null,
     isProcessing: false,
     uploadFileValue: null,
+    maxFileUploadSize: '8MB',
```

Comment:
Why is this hardcoded?

Reply:
Ooops

Reply:
I'd recommend remove the default value in frontend and show an error to user and ask to reload the page

Reply:
How do you want to show the error? @sukanya-rath